revpi_core.h

```diff
@@ -42,10 +42,14 @@ typedef struct _SRevPiProcessImage {
 		u8 i8uCPUFrequency;
 	} __attribute__ ((__packed__)) drv;	// 6 bytes
 	struct {
-		u8 i8uLED;
+		union {
+			u8 outputs;
+			u8 leds;
+		};
 		u16 i16uRS485ErrorLimit1;
 		u16 i16uRS485ErrorLimit2;
-	} __attribute__ ((__packed__)) usr;	// 5 bytes
+		u16 rgb_leds;
+	} __attribute__ ((__packed__)) usr;	// 5 to 7 bytes
```

Comment:
The comment is misleading, since the struct is now clearly 7 bytes in size. If we want to express, that in some cases only 5 of the 7 bytes are used, then this (and the reason why) should be explained in detail at this place.